lib/entities/webhook.ts

```diff
@@ -28,33 +28,36 @@ export type WebhookProps = {
   /**
    * Username for basic http auth
    */
-  httpBasicUsername: string
+  httpBasicUsername: string | null
 
   /**
    * Password for basic http auth
    */
-  httpBasicPassword: string
+  httpBasicPassword: string | null
 
   /**
    * Headers that should be appended to the webhook request
    */
-  headers: {
-    [key: string]: string
-  }
+  headers: Array<{ key: string; value: unknown; secret?: boolean }>
 
   /**
    * Topics the webhook wants to subscribe to
    */
   topics: string[]
 
+  /**
+   * Webhook filters
+   */
+  filters?: Array<{ [key: string]: any }>
```

Comment:
I'm fine with that but could be typed better. Will send a link to aschema

Reply:
updated